docs/docs/containers.md

```diff
@@ -41,3 +41,4 @@ to authenticate with the servers.
 | Wii U Chat ([authentication](https://github.com/PretendoNetwork/wiiu-chat-authentication) and [secure](https://github.com/PretendoNetwork/wiiu-chat-secure)) | Server for the Wii U Chat app.                                                                                                                                                      |
 | [BOSS](https://github.com/PretendoNetwork/BOSS)                                                                                                              | Server for BOSS (SpotPass) content.                                                                                                                                                 |
 | [Super Mario Maker](https://github.com/PretendoNetwork/super-mario-maker)                                                                                    | Server for Super Mario Maker online features.                                                                                                                                       |
+| [Minecraft: WiiU edition](https://github.com/PretendoNetwork/minecraft-wiiu)                                                                          | Server for Minecraft: WiiU Edition online features. ⚠️ By default Minigame's public matchmaking is disable by default. You can enable it by modifying the env variable `PN_MINECRAFT_ALLOW_PUBLIC_MATCHMAKING ` to 1 in the `environment/minecraft-wiiu.local.env` environment file.
```

Comment:
There isn't really a better way to make a warning inside a Markdown table. This is a suggestion based on my other review about the variable location and a minor wording improvement.

```suggestion
| [Minecraft: WiiU edition](https://github.com/PretendoNetwork/minecraft-wiiu)                                                                          | Server for Minecraft: WiiU Edition online features. ⚠️ For safety reasons, **public matchmaking is disabled by default**. You can enable it by setting the environment variable `PN_MINECRAFT_ALLOW_PUBLIC_MATCHMAKING ` to 1 in the `environment/minecraft-wiiu.env` file.
```